CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Security - Prevent remote environments creation with HTTP protocol
```

Comment:
[Conventions] Tiny detail :P 
```suggestion
- Security - Prevent remote environments creation with HTTP protocol.
```
(If we implement "conventional commits" I am pretty sure we wont have